docusaurus.config.js

```diff
@@ -87,7 +87,7 @@ module.exports = {
               className: "dropdown-unsupported-versions",
               value: "<b>Unsupported versions</b>",
             },
-            ...["1.2", "1.1", "1.0", "0.26", "0.25"].map((version) => ({
+            ...["1.3", "1.2", "1.1", "1.0", "0.26", "0.25"].map((version) => ({
```

Comment:
This array turns into the “unsupported versions” in the versions dropdown. It leads to some confusing information when viewing Optimize docs. 

We’ve customized the versions dropdown to display different _supported_ versions, depending on if you’re browsing the main docs or Optimize docs. This is done by [swizzling a component, and passing metadata to its children based on the current URL](https://github.com/camunda/camunda-platform-docs/blob/7b84279bba09848a4ef978af6164e21d1a1bf96d/src/theme/NavbarItem/index.js#L16-L18).

Unfortunately, the rendering of the static unsupported versions is a lot more simplistic than the rendering of the supported versions. Thus, when I’m viewing an Optimize doc, the static/unsupported versions show the _main docs_ versions, because that’s all that’s configured to show there. Like this: 

<img width="167" alt="image" src="https://github.com/camunda/camunda-platform-docs/assets/1627089/8305c3e2-b413-406e-ba90-9df5a0d5842e">

## How much effort do we want to put into this?

I am pretty sure I can swizzle one or two components to make that static list toggle between two different sets, depending on which docs instance you’re viewing. But I also wonder if this is necessary. This is somewhere between a “small” and “medium” amount of effort — closer to small, but there is risk that I can’t accomplish it. 

I can also think of at least two other solutions that are even smaller, but less correct. Since these are “unsupported” versions, we’ve already made decisions along the way guided by the idea that we don’t care about them as much as supported versions. Maybe they are good enough? 

## Possible solutions

1. A more complicated solution, in which we swizzle some components to make the “static list of unsupported versions” toggle based on the doc you’re viewing. This is a "small" to "medium" effort. I am unsure how the pre-Optimize-extraction versions should appear in this dropdown when viewing an Optimize page, if we go this route.
2. Just let the incorrect unsupported versions display when viewing an optimize doc (0 effort, what you see in my screenshot above).
3. Similar to the work for the upcoming release, we could display both the main version and the Optimize version in the dropdown, separated by a slash (e.g. `1.3 / 3.7.0`). This is a "small" effort. This is what it'd look like: 
   <img width="186" alt="image" src="https://github.com/camunda/camunda-platform-docs/assets/1627089/1ac1a181-181d-409f-81e3-f9538341bba4">
If we choose this option, the `/ 3.7.0` would also show in the unsupported versions when viewing a main docs page.

@akeller is there anything unclear here, and do you have any opinions on which route meets the "good enough" bar?

Reply:
I agree on spending a small amount of time/effort here. Spending no effort sounds like it will create confusion and may even block people from finding what they need. Ideally it doesn't because people are running one of the 3 supported versions, but we can't ding people for not being on our schedule.

I like your 3rd option, particularly because we have similar upcoming work. Showing in the main docs page feels natural.

Reply:
My most recent commits address this. Just for clarity (and I'll add this to the body of the PR), 

This is what the version dropdown looks like when viewing most docs:

![image](https://github.com/camunda/camunda-platform-docs/assets/1627089/9eb5e514-7322-46a9-8d54-db489063fc22)

This is what the version dropdown looks like when viewing an Optimize doc:

![image](https://github.com/camunda/camunda-platform-docs/assets/1627089/54285323-06bb-4fb5-b395-0fcd284e2e8c)